### DISCUSSION TITLE
missing files: _targets/meta/meta

### DISCUSSION THREAD

**Discussion by busmansholiday** [ropensci/targets#862] [General]
I am just starting with targets after your UserR2022 presentation, and I have opened a new Rstudio project, installed targets, run use_targest() and tried a simple  example, only to get this error: 
# tar_meta(fields = error, complete_only = TRUE)
Error:
! missing files: _targets/meta/meta

Given the importance of meta/meta, I am unclear about how I missed this step, and what I should do to repair it. 

Thanks in advance,

**Comment by wlandau** [comment]
Some functions require an already executed pipeline, and it looks like you may have used one of them. I recommend looking at https://books.ropensci.org/targets/walkthrough.html to get more familiar with the basics.
